scripts/openapi/gong/metadata/main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"log/slog"
+
+	"github.com/amp-labs/connectors/providers/gong/metadata"
+	"github.com/amp-labs/connectors/providers/gong/openapi"
+	"github.com/amp-labs/connectors/tools/fileconv/api3"
+	"github.com/amp-labs/connectors/tools/scrapper"
+	"github.com/getkin/kin-openapi/openapi3"
+)
+
+var ignoreEndpoints = []string{ // nolint:gochecknoglobals
```

Comment:
Curious, what is the current behavior if we try to use these endpoints with the connector?

Reply:
In general these endpoints are ignored because:
* There is no GET operation
* URL path is nested (this can be fixed and is thought of in other WIP PR), but I checked now that Gong doesn't seem to have any instance to fall into this category
* Requires query parameters which we don't support proxying
* Relies on one or many resource IDs
* Response returns single item instead of array.

I believe this covers all the reasons why URLs can be exempt.